medical-hub/src/component/Labtest/labtest.jsx

```diff
@@ -1,521 +0,0 @@
-import styled from 'styled-components'
-import { FeaturedLabs } from './FeaturedLabs'
-import { Slider } from './Slider'
-export  const LabTests = () => {
-    return (
-        <TimeStyled>
-            <div id='main'>
-                <div>
-                    <img src="https://onemg.gumlet.io/assets/9aaa4ea0-dcf1-11ec-a049-0273d826c612.png?format=auto" alt="img" />
-                </div>
-                <div>
-                    <div id='lab'>
-                        <span>Lab Test From The Comfort Of Your Home</span>
-                        <p>50,00,000+ lab tests booked | 20,00,000+ satisfied customers</p>
-                    </div>
-                    <div className='home'>
-                        <div>
-                            <img src="https://onemg.gumlet.io/Safe_38x38_labs_oe5ieq.png?format=auto" alt="img" />
-                            <p>100% Safe & Hygienic</p>
-                        </div>
-                        <div>
-                            <img src="https://onemg.gumlet.io/new_color_images/Lab_delivery_2x.png?format=auto" alt="img" />
-                            <p>Home Sample Pick Up</p>
-                        </div>
-                        <div>
-                            <img src="https://onemg.gumlet.io/new_color_images/Lab_online_report_2x.png?format=auto" alt="img" />
-                            <p>View Reports Online</p>
-                        </div>
-                        <div>
-                            <img src="https://onemg.gumlet.io/new_color_images/Lab_doctor_2x.png?format=auto" alt="img" />
-                            <p>Free Doctor Consultation</p>
-                        </div>
-                        <div>
-                            <img src="https://onemg.gumlet.io/new_color_images/Lab_offer_2x.png?format=auto" alt="img" />
-                            <p>Best Prices Guaranteed</p>
-                        </div>
-                    </div>
-                    <div className='package'>
-                        <button>VIEW POPULAR PACKAGES</button>
-                    </div>
-                </div>
-            </div>
-            <div id="popular">
-                <h3 className='heading'>Book A Lab Test</h3>
-                <div id="popular1">
-                    <div className='popular2'>
-                        <div>
-                            <div>
-                                <svg xmlns="http://www.w3.org/2000/svg" width="38" height="38" viewBox="0 0 38 38"><g fill="none" fill-rule="evenodd" transform="translate(1 1)"><circle cx="18" cy="18" r="18" fill="#FFFCF8" stroke="#FFFFFF"></circle><g stroke="#000" stroke-linecap="round" stroke-linejoin="round"><path fill="#FFF" d="M10.945 17.814v7.067c0 .721.574 1.309 1.284 1.309h4.11v-5.23h3.258v5.23h4.124c.71 0 1.286-.588 1.286-1.309v-7.067m1.943 0L17.975 9.05 9 17.814"></path><path d="M25.146 13.268v-1.655c0-.722-.574-1.309-1.285-1.309H22.35"></path></g></g></svg>
```

Comment:
define all the svgs elsewhere and then import them here. it is making a lot of clutter here